picard/tagger.py

```diff
@@ -1193,36 +1243,28 @@ def longversion():
 
 
 def print_help_for_commands():
+    maxwidth = 80
+    helpcmd = []
+    for name in sorted(REMOTE_COMMANDS):
+        remcmd = REMOTE_COMMANDS[name]
+        s = "  - %-34s %s" % (name + " " + remcmd.help_args, remcmd.help_text)
+        helpcmd.append(fill(s, width=maxwidth, subsequent_indent=' '*39))
+
     print("""usage: picard -e [command] [arguments ...]
-or
-picard -e [command 1] [arguments ...] -e [commands 2] [arguments ...]
+    or picard -e [command 1] [arguments ...] -e [command 2] [arguments ...]
 
 List of the commands available to execute in Picard from the command-line:
+""")
+    print("\n".join(helpcmd))
+
+    def fmt(s):
+        print(fill(s, width=maxwidth, initial_indent=' '*2))
 
-  - CLUSTER                           cluster all files in the cluster pane
-  - FINGERPRINT                       calculate acoustic fingerprints for all (matched) files in
-                                      the album pane
-  - LOOKUP                            lookup all clusters in the cluster pane.
-  - LOOKUP_CD [device/log file]       read CD from the selected drive and lookup on MusicBrainz.
-  - QUIT                              exit the running instance of Picard
-  - REMOVE [absolute path (1 file)]   remove the file from Picard
-  - REMOVE_ALL                        remove all files from Picard
-  - REMOVE_SAVED                      remove all saved releases from the album pane
-  - SAVE_COMPLETE                     save all completely matched releases in the album pane
-  - SCAN                              scan all files in the cluster pane
-  - SHOW                              make the running instance the currently active window
-  - SUBMIT_FINGERPRINTS               submit outstanding acoustic fingerprints for all (matched)
-                                      files in the album pane
-
-- Commands are case insensitive (not args!), i.e. SHOW == show == ShOw
-- If there is no running instance, picard -e blocks Picards startup and does nothing
-- If there is an existing instance, picard -e will send all the positional arguments in the given order.
-  Then the commands. So the files passed to the new instance with a command will be processed as well.
-- Arguments are optional and not providing them will not cause Picard to crash
-  - LOOKUP_CD without the argument will default to the first (alphabetically) available disc drive
-  - REMOVE without the argument will not do anything
-  - other commands ignore the arguments and act the same regardless of their amount
-- Only one path can be send per one REMOVE command""")
+    fmt("Commands are case insensitive.")
+    fmt("If there is no running instance, picard -e blocks Picards startup and does nothing.")
+    fmt("If there is an existing instance, picard -e will send all the positional arguments in the given order."
+    "Then the commands. So the files passed to the new instance with a command will be processed as well.")
```

Comment:
```suggestion
    fmt("If there is an existing instance, picard -e will send all the positional arguments in the given order "
    "followed by the commands, so the files passed to the new instance with a command will be processed as well.")
```

I still find this a bit confusing, because I don't think it makes it clear that the files passed with a command will be processed as part of that command and NOT as files to add.

Reply:
But these files will be processed as files to add. What I was trying to say is that they will be added before the command execution, so they might be affected by it.

Reply:
That's good.  In that case we need to make it clear that files to be added appear on the command line BEFORE any `-e command` options (or else the files to be added will appear as arguments to the specified `-e` command.